projects/dsp-ui/src/lib/viewer/values/interval-value/interval-input/interval-input.component.html

```diff
@@ -11,4 +11,10 @@
       End is <strong>required</strong>.
     </mat-error>
   </mat-form-field>
+
+  <div class="date-form-error">
+    <mat-error *ngIf="((startIntervalControl.hasError('startEndSameTypeRequired') || endIntervalControl.hasError('startEndSameTypeRequired')))">
```

Comment:
Is there a need for null checks for start or end as in case of the date?
Or is this taken care of by `startEndSameTypeValidator`?

Reply:
this is handled in `startEndSameTypeValidator`